packages/adapter/lib/adapter/adapter.js

```diff
@@ -5516,7 +5518,7 @@ function Adapter(options) {
 
     // initStates is called from initAdapter
     const initStates = cb => {
-        logger.debug(this.namespaceLog + ' objectDB connected');
+        logger.silly(this.namespaceLog + ' objectDB connected');
```

Comment:
I would like to keep this as debug because it is good to know that the db was connected successfully
```suggestion
        logger.debug(this.namespaceLog + ' objectDB connected');
```